src/OrleansZooKeeperUtils/ZooKeeperHostingExtensions.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Orleans.Runtime.Membership;
+using OrleansZooKeeperUtils.Configuration;
+
+namespace Orleans.Hosting
+{
+    public static class ZooKeeperHostingExtensions
+    {
+        /// <summary>
+        /// Configure siloHostBuilder with ZooKeeperMembershipTable
+        /// </summary>
+        /// <param name="builder"></param>
+        /// <param name="configureOptions"></param>
+        /// <returns></returns>
+        public static ISiloHostBuilder UseZooKeeperMembershipTable(this ISiloHostBuilder builder,
```

Comment:
these methods suffixed with `Table` but the overloads for IServiceCollection are not. I assume the should be the same?